test/browser.js

```diff
@@ -24,8 +23,8 @@ async function createAndLoadRepo () {
   const store = new Datastore(dir, { extension: '', createIfMissing: true })
   await store.open()
 
-  await store.put(VERSION_KEY, Buffer.from(loadFixture('test/fixtures/test-repo/version')))
-  await store.put(CONFIG_KEY, Buffer.from(loadFixture('test/fixtures/test-repo/config')))
+  await store.put(VERSION_KEY, loadFixture('test/fixtures/test-repo/version'))
+  await store.put(CONFIG_KEY, loadFixture('test/fixtures/test-repo/config'))
```

Comment:
🤷‍♂️ I'm not sure I get why `Buffer.from` was used in first place because `loadFixture` returns buffer as far as I can tell. It's probably a good idea to turns them into `Uint8Array`s to ensure that put does in fact works with `Uint8Array` as opposed to `Buffer`.

Reply:
I'm not sure either TBH.

Converting them to `Unint8Array`s to ensure it works seems unnecessary as we're not testing the store implementation here.